app/controllers/unattended_controller.rb

```diff
@@ -217,8 +217,14 @@ def update_ip
     logger.debug "Built notice from #{ip}, current host ip is #{@host.ip}, updating" if @host.ip != ip
```

Comment:
This isn't taking IPv6 into account and needs to be moved into the if block's branches.